cobc/parser.y

```diff
@@ -15239,15 +15239,35 @@ inspect_before:
   {
 	$$ = CB_BUILD_FUNCALL_1 ("cob_inspect_before", $3);
   }
+ | BEFORE _initial inspect_or_list
+  {
+	cb_verify(cb_inspect_or, _("INSPECT or"));
```

Comment:
that string does not need a translation as both INSPECT and OR reference keywords; you possibly can improve that string in any case